### PR TITLE
lightning_classy_vision: add model interpretability example

### PR DESCRIPTION
**Pull request by d4l3k**
This is a model interpretability example. It loads the model from a training step and then runs integrated gradients on it via captum (https://captum.ai/tutorials/CIFAR_TorchVision_Interpret) and then writes out 5 samples to the output path.

Test plan:

```
cd examples/apps && torchx run --scheduler local lightning_classy_vision/component.py:interpret --image (pwd) --output_path /tmp/output --data_path /tmp/data.tar.gz --load_path /tmp/model-out/last.ckpt
scripts/kfpint.py
```

http://5ab6bab9-istiosystem-istio-2af2-1926929629.us-west-2.elb.amazonaws.com/_/pipeline/?ns=torchx-dev#/runs/details/0a4a26f2-e1b2-4b99-be4e-3c0d74875770

### CODE REVIEW

File: torchx/pipelines/kfp/adapter.py [hunk 1]
```diff
@@ -216,7 +216,7 @@ def factory_wrapper(*args: object, **kwargs: object) -> dsl.ContainerOp:
             mem_str = f"{int(mem)}M"
             container.set_memory_request(mem_str)
             container.set_memory_limit(mem_str)
-        if (gpu := resources.gpu) >= 0:
+        if (gpu := resources.gpu) > 0:
```
Comment (d4l3k) [inline torchx/pipelines/kfp/adapter.py:219]: this is needed since KFP throws an error if you ask for 0 GPUs
